### PR TITLE
Migrate ACR tests

### PR DESCRIPTION
**Pull request by lmolkova**

### CODE REVIEW

File: sdk/containerregistry/azure-containers-containerregistry-perf/pom.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.12.2</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <version>1.13.0-beta.2</version> <!-- {x-version-update;beta_com.azure:azure-identity;dependency} -->
```
Comment (g2vinay) [inline sdk/containerregistry/azure-containers-containerregistry-perf/pom.xml:61]: we can now change this to stable release of 1.13.0

File: sdk/containerregistry/azure-containers-containerregistry/pom.xml [hunk 2]
```diff
@@ -129,7 +129,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.12.2</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <version>1.13.0-beta.2</version> <!-- {x-version-update;beta_com.azure:azure-identity;dependency} -->
```
Comment (g2vinay) [inline sdk/containerregistry/azure-containers-containerregistry/pom.xml:132]: same

**Review by g2vinay — APPROVED** [review]

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**